src/capi.ts

```diff
@@ -101,12 +101,21 @@ const includesTweets = (content: Content): boolean => {
 // ----- Functions ----- //
 
 const capiEndpoint = (articleId: string, key: string): string => {
+    // If you need a new field here, MAPI probably also needs updating
```

Comment:
I wonder if longer term this is something that could live in the `types` repo and be shared with MAPI?

Reply:
yeah that would be a good idea, otherwise it would be easy for them to get out of sync